app/views/idv/welcome/show.html.erb

```diff
@@ -1,15 +1,13 @@
 <% self.title = @title %>
```

Comment:
I think here we could call `@presenter.title` and skip setting the ivar in the controller.

Reply:
We can here, but we still have to set it in the controller, or else the Rails base layout complains.

Reply:
I stand corrected.